modules/integration/tests-integration/tests-es/src/test/java/org/wso2/carbon/registry/es/notifications/RestServiceStoreNotificationTestCase.java

```diff
@@ -0,0 +1,194 @@
+/*
+*Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+
+package org.wso2.carbon.registry.es.notifications;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.wink.client.ClientResponse;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.annotations.*;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.greg.integration.common.utils.GREGIntegrationBaseTest;
+import org.wso2.greg.integration.common.utils.GenericRestClient;
+
+import javax.ws.rs.core.MediaType;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.testng.Assert.assertNotNull;
+
+/**
+ * This class testes subscription & notification for rest services on store notification
+ */
+public class RestServiceStoreNotificationTestCase extends GREGIntegrationBaseTest {
+
+    private static final Log log = LogFactory.getLog(RestServiceStoreNotificationTestCase.class);
+
+    private TestUserMode userMode;
+    String jSessionIdPublisher;
```

Comment:
Hi Fernando,
Thanks for pointing this to me. This variable has been already moved to a local variable in PR #343 
But I have identified few other places where there are default access mode. I Updated the PR #343 with the fixes.

Thanks!